client/src/globals.js

```diff
@@ -1,8 +1,30 @@
+import * as R from 'ramda'
 
 let users = { }
+let userCallbackCouter = 0
```

Comment:
typo

Reply:
additonally, you can just return `userChangeCallbacks.size` instead of keeping track

Reply:
ah they're used as id